modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} clusters with the {lcao}
+
+You can upgrade the platform version of the managed {sno} clusters with the {lcao}.
+By using generated a seed image, this upgrade process allows you to skip multiple minor releases and it is faster than the standard upgrade method.
+
+You generate the seed image from a cluster that is upgraded and configured with the desired {product-title} and applications by using the standard upgrade method. This is the seed cluster.
+
+[IMPORTANT]
+====
+The image-based upgrade uses custom images that are specific to the hardware platform the clusters are running on.
+Each different hardware platform requires a separate custom image.
+====
+
+With this seed image, you can upgrade the platform version on any {sno} that matches the image's cluster profile and then restore applications.
+
+//How do we specify the hardware exactly? Is it baked into the seed image based on the seed cluster we're using?
+
+The {lcao} uses two custom resources (CRs):
+
+* The `SeedGenerator` CR specifies the platform version of the seed cluster that you want to use to upgrade the target clusters.
+* The `imageBasedUpgrade` CR specifies the generated seed image's location and you use it to step through the upgrade stages.
+
+After generating the seed image, the upgrade process has four stages:
+
+* `Idle`
+* `Prep`
+* `Upgrade`
+* `Rollback` (Optional)
+
+[discrete]
+== Idle stage
+
+There is no ongoing upgrade and the cluster is ready to move to the `Prep` stage.
+
+[discrete]
+== Prep stage
+
+You can complete this stage in advance.
+
+The {lcao} ensures that the node is ready to proceed to the `Upgrade` stage. This includes the following steps:
+
+* Installing and configuring the OADP Operator
```

Comment:
As far as I know, the OADP operator must be installed by the user previously cc @donpenney @browsell . It can be installed in many ways depending on the cluster, but in our case where we foresee a fleet of clusters being managed by RHACM, the ZTP/TALM creating the proper policies makes more sense to me.

Reply:
it is a prerequisite that the oadp operator is installed on the target cluster prior to starting this workflow